src/api/layer/authenticated_http.ts

```diff
@@ -114,6 +114,7 @@ const handleResponse = async <Return>(res: Response) => {
     )
     throw apiError
   }
+  if (res.status === 204) return null as Return
```

Comment:
null feels slightly weird to me here, do we return this if an account is already deleted?